src/main/java/nl/novi/techiteasymodel/controllers/TelevisionController.java

```diff
@@ -0,0 +1,41 @@
+package nl.novi.techiteasymodel.controllers;
+
+import nl.novi.techiteasymodel.models.Television;
+import nl.novi.techiteasymodel.repositories.TelevisionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+import java.util.ArrayList;
+
+@RestController
+@RequestMapping("/televisions")
+public class TelevisionController {
+
+    @Autowired
+    private TelevisionRepository repos;
+
+    private ArrayList<Television> televisions;
+
+
+    @GetMapping("")
+    public ResponseEntity<String> getTelevisions(){
+        return ResponseEntity.ok("television" + repos.findAll());
```

Comment:
Volgens mij geef je met alleen `ok(televisions)` geen televisies terug. 
```suggestion
         return ResponseEntity.ok().body(repos.findAll();
```